charts/greptimedb-cluster/templates/cluster.yaml

```diff
@@ -286,7 +286,12 @@ spec:
         {{- if .Values.meta.podTemplate.main.image }}
         image: {{ .Values.meta.podTemplate.main.image }}
         {{- end }}
-        {{- if .Values.meta.podTemplate.main.command }}
+        {{- if or .Values.license.enabled }}
```

Comment:
Suggestion to: `{{- if or .Values.license.enabled .Values.meta.podTemplate.main.command }}`